src/main/java/com/spartronics4915/frc2020/subsystems/PanelRotator.java

```diff
@@ -1,16 +1,47 @@
 // I'm bad at naming things. Please come up with a better name...
 package com.spartronics4915.frc2020.subsystems;
 
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ColorSensorV3;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.spartronics4915.frc2020.Constants;
 import com.spartronics4915.lib.subsystems.SpartronicsSubsystem;
 
+import 
+
+import edu.wpi.first.wpilibj.DigitalInput;
+
 public class PanelRotator extends SpartronicsSubsystem {
+    private final CANSparkMax mSpinMotor;
```

Comment:
None of these should be final (we modify them in the constructor).

Reply:
Actually this is a best practice (good job, Noah). Marking these as final is OK as long as they are only assigned once (in the constructor.)

If you mark them as final this way then the compiler will warn you if you haven't initialized the variables in the constructor.